modules/ROOT/pages/collaboration.adoc

```diff
@@ -40,7 +40,23 @@ date: String
 status: String
 --
 
-In this example, your default query method is `ordersById`, and its primary key is `orderId`.`ordersById` retrieves the Order type and uses `orderId` to retrieve specific information based on that ID field. Note that the data type of the default query method’s input argument ID must be the same as the data type of the primary key. In this example, both are of data type `ID!`.
+In this example, the default query method is `ordersById`, and its primary key is `orderId`.`ordersById` retrieves the `Order` type and uses `orderId` to retrieve specific information based on the `ID` field. Note that the data type of the default query method’s input argument `ID` must be the same as the data type of the primary key. In this example, both are of data type `ID!`.
+
+=== Default Query Method with Composite Keys
+
+In the following example, using composite keys, the default query method is `productDetail`, and its composite keys are `productId` and `productSku`. Both retrieve the `Product` type and use `productId` and `productSku` to retrieve specific information based on the `string` fields. Note that the data type of the default query method’s input arguments must be the same as the data type of each primary key. In this example, both are of data type `string!`.
+
+[source]
+--
+productDetail (id: string!, sku: string!): Product
+--------------------
+--------------------
+Product
+--------------------
+productDetail: string!, string! 
```

Comment:
This `string` must be `String` because is a Type like as the next fields

Reply:
fixed. thanks.